test/RoaveTest/SecurityAdvisories/VersionConstraintTest.php

```diff
@@ -136,15 +130,6 @@ public function testContainsWithMatchingRanges() : void
         self::assertFalse($this->callContains($constraint2, $constraint1));
     }
 
-    public function testCannotCompareComplexRanges() : void
```

Comment:
Removals in tests are shady: what changed? Why are these no longer working/supported?